File: week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //get size of grid
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    //Print # and . and " "
    //Loop for rows
    for (int i = 0; i < n; i++)
    {
        //Inner loop for . and " "
        for (int x = 0; x < n - i - 1; x++)
        {
            printf(".");
        }
        //Inner Loop for #
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        //Inner loop for inside ../space
        for (int k = 0; k < 2; k++)
        {
            printf(" ");
        }
        //Inner loop for second #
        for (int l = 0; l <= i; l++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: week6/sentimental-mario-more/mario.py


def main():
    height = get_height()
    # printing rows

    for i in range(height):
        # printing inner loop
        x = int(height - i - 1)
        for j in range(x):
            print(" ", end="")

        # inner loop with #
        y = int(height - x)
        for k in range(y):
            print("#", end="")

        # inner loop for inside space
        for l in range(2):
            print(" ", end="")

        # inner loop for second #
        for m in range(y):
            print("#", end="")
        print()


# get size of grid
def get_height():
    while True:
        try:
            n = int(input("Height: "))
            if n > 0 and n < 9:
                return n
        except ValueError:
            print("Error")


main()